.github/CODEOWNERS

```diff
@@ -6,11 +6,23 @@
 /.github/         @maximedenes
 # Secondary maintainer @Zimmi48
 
+########## Build system ##########
+
+/Makefile*        @gares
+
+/configure*       @ejgallego
+
+/META.coq*        @ejgallego
```

Comment:
The filter has changed for this file. Is this done on purpose?

Reply:
This is a mistake. Will fix.